tests/test_calc_ab_metrics.py

```diff
@@ -12,32 +12,10 @@
     calc_ab_metrics,
     calculate_metrics_data,
     create_record_diff_matrix_dataset,
-    generate_field_diff_bools_for_record,
 )
 from abdiff.core.utils import load_dataset, read_run_json
 
 
-def test_record_field_diffs_no_diffs():
-    diff_data = {}
-    assert generate_field_diff_bools_for_record(diff_data) == {}
-
-
-def test_record_field_diffs_one_diff():
-    diff_data = {"color": "green"}
-    assert generate_field_diff_bools_for_record(diff_data) == {"color": 1}
-
-
-def test_record_field_diffs_diff_from_inserts_and_deletes_counted_only_once():
-    diff_data = {
-        "$insert": {"fruits": "strawberry"},
-        "$delete": {"vegetables": "onion"},
-    }
-    assert generate_field_diff_bools_for_record(diff_data) == {
-        "fruits": 1,
-        "vegetables": 1,
-    }
-
-
```

Comment:
"Cutting is delivering" is my new favorite motto

Reply:
Hahahahah, agreed!  No need for veggies.